config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+  resources :users do 
+    resources :posts
```

Comment:
- [ ] Please do not expose unnecessary endpoints, for now, it is a requirement to have just index and show exposed for both, users and posts, please do so to comply with the project requirements, everything else is good to go 😉